src/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint global-require: 0 */
```

Comment:
Interesting again, in my other code base, it doesn't complain about use of `` require ``. I'll investigate into that.